src/chat/chat_history.rs

```diff
@@ -73,13 +73,6 @@ impl Widget for ChatHistory {
     fn handle_event(&mut self, cx: &mut Cx, event: &Event, scope: &mut Scope) {
         self.deref.handle_event(cx, event, scope);
         self.widget_match_event(cx, event, scope);
-
-        // TODO This is a hack to redraw the chat history and reflect the
-        // name change on the first message sent.
-        // Maybe we should send and receive an action here?
-        if let Event::Signal = event {
-            self.redraw(cx);
-        }
```

Comment:
Note: Idk why, but removing this didn't cause any issues. Am I missing something?

Reply:
Also seems unnecessary in my tests, I'd say safe to remove!